server/src/test/java/io/crate/analyze/expressions/ExpressionAnalyzerTest.java

```diff
@@ -369,7 +369,7 @@ public void testParameterExpressionInLikeAny() throws Exception {
 
     @Test
     public void testAnyWithArrayOnBothSidesResultsInNiceErrorMessage() {
-        expectedException.expectMessage("Cannot cast `xs` of type `integer_array` to type `integer`");
+        expectedException.expectMessage("unknown function: any_=(integer_array, integer_array)");
```

Comment:
Do you think it would be worth following up on this to format the function name in a way that is closer to what the user entered? So that we'd get something like `Unknown function: integer_array = ANY(integer_array)`

Reply:
Yes good point. Will follow up here dedicated or do you prefer to add it to this PR?

Reply:
Follow up sounds good